### PR TITLE
feat(wallet): FeatureCards component

### PR DESCRIPTION
**Pull request by DiogoSoaress**
## What it solves
Implements the Featured Cards section.

## Figma
https://www.figma.com/design/qtXXlwdslAze6yNbLLzIRa/Website-redesign-2.0?node-id=15367-15363&node-type=frame&t=DLYMrseygMcxi70x-0

Screenshot 2024-10-31 at 16 57 58

### CODE REVIEW

File: src/components/Wallet/FeatureCards/index.tsx [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+          <Typography variant="h2">{title}</Typography>
+        </Grid>
+
+        <Grid item xs={12} md={5} alignContent="flex-end">
+          <Typography variant="h5">{text}</Typography>
+        </Grid>
+      </Grid>
+
+      <Grid container mt={{ xs: '50px', md: '100px' }} columnSpacing="30px" rowGap="30px">
+        {restItems.reverse().map((item, index) => {
```
Comment (iamacook) [inline src/components/Wallet/FeatureCards/index.tsx:49]: You can save the double reverse by using `slice`:
```suggestion
  const lastItem = items[items.length - 1]
  const restItems = items.slice(0, -1)

  return (
    <Container className={layoutCss.containerMedium}>
      <Grid container justifyContent="space-between" columnSpacing="30px" rowGap="30px">
        <Grid item xs={12} md={7}>
          <Typography variant="h2">{title}</Typography>
        </Grid>

        <Grid item xs={12} md={5} alignContent="flex-end">
          <Typography variant="h5">{text}</Typography>
        </Grid>
      </Grid>

      <Grid container mt={{ xs: '50px', md: '100px' }} columnSpacing="30px" rowGap="30px">
        {restItems.map((item, index) => {
```

File: src/content/wallet.json [hunk 2]
```diff
@@ -160,6 +160,57 @@
...
+        "caption": "Trade seamlessly",
+        "title": "Swap like a pro",
+        "text": "with MEV protection and security, ensuring your intent-based transactions will not fail.",
+        "image": {
+          "src": "/images/Wallet/Features/trade.png",
+          "alt": "Token swap modal"
+        }
+      },
+      {
+        "caption": "Optimise",
```
Comment (iamacook) [inline src/content/wallet.json:180]: I think we adhere to American English.

```suggestion
        "caption": "Optimize",
```

File: src/content/wallet.json [hunk 3]
```diff
@@ -160,6 +160,57 @@
...
+        "title": "Explore Safe Apps",
+        "text": "Swapping, staking, NFTs, identity and more. Interact with 200+ apps integrated with Safe.",
+        "link": {
+          "title": "View ecosystem projects",
+          "href": "https://safe.global/ecosystem",
+          "variant": "link"
+        },
+        "image": {
+          "src": "/images/Wallet/Features/safe-apps.png",
+          "alt": "Safe apps logos"
```
Comment (iamacook) [inline src/content/wallet.json:209]:
```suggestion
          "alt": "Safe Apps logos"
```

File: src/components/Wallet/FeatureCards/index.tsx (hunk 1 above)
Comment (DiogoSoaress) [reply src/components/Wallet/FeatureCards/index.tsx]: Good suggestion!

**Review by iamacook — APPROVED** [review]

**Pull request merged**